fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/metadata/KeyExpressionTest.java

```diff
@@ -766,6 +768,21 @@ void testSplitConcat() {
                 evaluate(splitConcat, numbers));
     }
 
+    @Test
+    void testCanProperlyGetValueExpressionFromKeyWithValue() {
```

Comment:
I think "properly" is a little vague here. Perhaps "single value" would be a better term